Web/src/utils/animations.ts

```diff
@@ -0,0 +1,25 @@
+import { keyframes } from 'styled-components';
+
+export const blinkRed = keyframes`
+  0% {
+    background: rgba(var(--red-rgb), 0);
+  }
+  50% {
+    background: rgba(var(--red-rgb), 0.6);
```

Comment:
50% is equivalent to 0.6? 🤔

Reply:
0.6 is the opacity, 50% is the proportion of the total time the keyframe is at 😄

Reply:
Oops you're right sorry I derped hahah